packages/components/cascader/src/cascader.vue

```diff
@@ -342,10 +355,14 @@ const cascaderKls = computed(() => {
   ]
 })
 
+const iconComponent = computed(() =>
+  props.suffixIcon ? props.suffixIcon : ArrowDown
+)
+
 const cascaderIconKls = computed(() => {
   return [
     nsInput.e('icon'),
-    'icon-arrow-down',
+    'icon-down',
```

Comment:
I would suggest that we add a redundant class for this instead of removing it, removing this is considered a breaking change which is not what we want right now.